src/Database/Eloquent/Concerns/QueriesRelationships.php

```diff
@@ -0,0 +1,69 @@
+<?php namespace October\Rain\Database\Eloquent\Concerns;
+
+use Illuminate\Database\Eloquent\Concerns\QueriesRelationships as IlluminateQueriesRelationships;
+use Illuminate\Support\Str;
+
+trait QueriesRelationships
+{
+    use IlluminateQueriesRelationships {
+        IlluminateQueriesRelationships::withCount as IlluminateWithCount;
```

Comment:
@Klaasie is there any need for this alias, given that you don't use it within your new `withCount` method?

Reply:
I guess given the circumstances the alias doesn't make a lot of sense. I thought adding the alias would provide developers with the opportunity to use the Illuminate version of the method if the need may be.

But since there's no difference in results and this actually fixes a potential error I guess it could be left out. Would you want me to make that modification?

Reply:
Yep, I reckon just remove it. :)

Reply:
Done!